11/09/2022 18:26:11 - INFO - __main__ - ----------------------------------------------------------------------------------------------------
11/09/2022 18:26:11 - INFO - __main__ - Training models on Vision-Language continual learning tasks...
11/09/2022 18:26:11 - INFO - __main__ - ----------------------------------------------------------------------------------------------------
11/09/2022 18:26:11 - INFO - __main__ - ********************** found the task token with same task key! *****************************
11/09/2022 18:26:11 - INFO - __main__ - Training vilt model on task #1: SNLI-VE
Creating DyTox!
Traceback (most recent call last):
  File "/home1/caiyulia/.conda/envs/climb/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home1/caiyulia/.conda/envs/climb/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/project/rostamim_919/caiyulia/Multi-Dytox/src/run.py", line 286, in <module>
    main()
  File "/project/rostamim_919/caiyulia/Multi-Dytox/src/run.py", line 209, in main
    task_trainer = task_trainer_class(args, task_configs, model_config, device, teacher_model, num_task)
TypeError: __init__() takes 6 positional arguments but 7 were given
